runtime/cere/src/lib.rs

```diff
@@ -159,7 +159,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 73108,
+	spec_version: 731089,
```

Comment:
Looks like you incorrectly increased the version

Reply:
sorry .. resolved